buildscripts/travis-build.sh

```diff
@@ -20,8 +20,7 @@ function checkGitDiff() {
 	if [[ `git diff --shortstat | wc -l` != 0 ]]; then echo "Some files got changed after $1";printf "\n";git diff;printf "\n"; exit 1; fi
 }
 
-#make golint-travis
```

Comment:
Why is golint disabled?

Reply:
forget to revert the change earlier , reverted now

Reply:
We don't have Makefile target for lint currently , so let it as same as ealier ,
later we can add `golangci-lint` for better linting support.